applicationinsights.json

```diff
@@ -32,11 +32,6 @@
               "key": "http.url",
               "value": "https?://[^/]+/health.*",
               "matchType": "regexp"
-            },
-            {
-              "key": "http.status_code",
-              "value": "200",
```

Comment:
tried both a string here and a number, neither seem to work.
According to https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-sampling-overrides it should be able to be set like this, but just causes it to be ignored